src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -1292,14 +1298,28 @@ void MainWindow::slot_open_recent()
     }
 }
 
-void MainWindow::slot_clear_open_recent_files_menu()
+void MainWindow::slot_clear_recent_files_menu()
 {
     QSettings settings(SETTINGS_ORGANIZATION, SETTINGS_APPLICATION);
     settings.setValue("recent_file_list", QStringList());
 
     update_recent_files_menu(QStringList());
 }
 
+void MainWindow::slot_clear_recent_files_nonexistent()
+{
+    QSettings settings(SETTINGS_ORGANIZATION, SETTINGS_APPLICATION);
+    QStringList files = settings.value("recent_file_list").toStringList();
+    QStringList existentFiles;
```

Comment:
We use snake_case for variables (detailed coding guidelines: https://github.com/appleseedhq/appleseed/wiki/Coding-Philosophy-and-Guidelines).

Reply:
Oh right, I forgot